#Top-left corner of the bounding box
geometry_msgs/Point position

#The width of the bounding box
float64 width
#The height of the bounding box
float64 height